### PR TITLE
Styled register form and login

### PR DESCRIPTION
**Pull request by annikavennemann**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -2,42 +2,17 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>Ohhhboarding</title>
```
Comment (FabianAlleblas) [inline frontend/public/index.html:12]: Oh 😄

File: frontend/src/components/Registration/Form.js [hunk 2]
```diff
@@ -11,57 +14,69 @@ export default function Form({ onSubmit }) {
...
                 
-                <label htmlFor="startdate">Start date</label>
-                <input
-                    id="startdate"
-                    type="date"
-                    name="startdate"
-                    onChange={onChange}
-                    value={userData.startdate}
-                />
+                <StartdateStyled htmlFor="startdate" >
```
Comment (miriamhaenle) [inline frontend/src/components/Registration/Form.js:58]: Auch hier könnte einfach `Startdate` reichen

File: frontend/src/components/Registration/Form.js [hunk 3]
```diff
@@ -11,57 +14,69 @@ export default function Form({ onSubmit }) {
...
+                    <button onClick={registerUser}><img src={signUpButton} alt=""/></button>
+                </SignUpWrapper>
 
-                <button onClick={registerUser}>Register User</button>
+            </FormStyled>
 
-            </form>
-            <p>You already have an account yet?</p>
-          <a href="/">Sign in!</a>
+            <FooterStyled>
```
Comment (miriamhaenle) [inline frontend/src/components/Registration/Form.js:76]: `Footer`

File: frontend/src/App.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ export default function App() {
 
   return (
     <Router>
-      <main>
+      <>
```
Comment (FabianAlleblas) [inline frontend/src/App.js:16]: You could import and use the Browserrouter in the index.js

File: frontend/src/components/Registration/Form.js [hunk 5]
```diff
@@ -74,23 +89,116 @@ export default function Form({ onSubmit }) {
...
-            headers: myHeaders,
-            body: raw,
-            redirect: 'follow',
-        };
-    
-        fetch('http://onboarding.local/user', requestOptions)
+
+        if (validRegistration(userData)){
+            onSubmit(userData)
+            var myHeaders = new Headers();
```
Comment (miriamhaenle) [inline frontend/src/components/Registration/Form.js:95]: Gibt es einen bestimmten Grund, dass du hier `var` verwendest anstatt `let` oder `const`?

**Review by miriamhaenle — COMMENTED** [review]
Nice job, code looks very clean and straightforward. 😃

File: frontend/src/components/Registration/Form.js [hunk 6]
```diff
@@ -11,57 +14,69 @@ export default function Form({ onSubmit }) {
     
     return (
         <>
-            <form onSubmit={registerUser}>
-                <label htmlFor="firstname">First name</label>
+            <header>
+                <HeaderImg src={signUpHeader} alt="Welcome to Ohhhboarding!" />
+            </header>
+            <FormStyled onSubmit={registerUser}>
```
Comment (FabianAlleblas) [inline frontend/src/components/Registration/Form.js:20]: I would remove the Styled, just Form should be fine

File: frontend/src/components/Registration/Form.js (hunk 2 above)
Comment (FabianAlleblas) [inline frontend/src/components/Registration/Form.js:58]: same as Form, maybe something with label?

File: frontend/src/components/Registration/Form.js (hunk 3 above)
Comment (FabianAlleblas) [inline frontend/src/components/Registration/Form.js:76]: Footer should nbe enough

File: frontend/src/components/Registration/Form.js [hunk 7]
```diff
@@ -74,23 +89,116 @@ export default function Form({ onSubmit }) {
...
+                headers: myHeaders,
+                body: raw,
+                redirect: 'follow',
+            }
+
+            fetch('http://onboarding.local/user', requestOptions)
             .then((response) => response.text())
             .then(result => console.log(result))
-            .catch((error) => console.log('error', error));
+            .catch((error) => console.log('error', error))
```
Comment (FabianAlleblas) [inline frontend/src/components/Registration/Form.js:109]: Would consider some error handling if something went wrong during the request

File: frontend/src/components/Registration/Form.js [hunk 8]
```diff
@@ -74,23 +89,116 @@ export default function Form({ onSubmit }) {
...
+    margin: 0;
+  }
+
+  button {
+    background-color: transparent;
+    border: none;
+  }
+`
+
+const FooterStyled = styled.footer`
```
Comment (FabianAlleblas) [inline frontend/src/components/Registration/Form.js:186]: Is this the same footer as in the Login.js? If so, maybe you could make a footer component, if you also want to use it on other pages :)

**Review by FabianAlleblas — COMMENTED** [review]
Nice one 😃 , I left a few minor comments, I hope they help :).

File: frontend/src/components/Registration/Form.js (hunk 5 above)
Comment (annikavennemann) [reply frontend/src/components/Registration/Form.js:95]: Nein, das ist vermutlich durch Postman entstanden - ich ändere das um. Danke :)

File: frontend/src/components/Registration/Form.js (hunk 8 above)
Comment (annikavennemann) [reply frontend/src/components/Registration/Form.js:186]: Well yes it has the same structure on the form- and login-page but besides that I won't use it anymore. Would you still make it a component though? :)

File: frontend/src/components/Login/Login.js [hunk 9]
```diff
@@ -9,29 +12,38 @@ export default function Login() {
 
     return (
         <>
-          <form>
-            <label htmlFor="email">Email</label>
+          <header>
+              <HeaderImg src={signInHeader} alt="Ohhh hi! Great day!" />
+          </header>
+          <FormStyled>
```
[the remark this answers is not in the document]
Comment (annikavennemann) [reply frontend/src/components/Login/Login.js:18]: Allright, danke fürs die review :)

**Pull request merged**